src/scielo/bin/xml/app_modules/app/data/kernel_document.py

```diff
@@ -0,0 +1,55 @@
+from ...generics import fs_utils
+from . import scielo_id_gen
+import xml.etree.ElementTree as ET
+
+
+def add_article_id_to_received_documents(
+    issn_id, year_and_order, received_docs, registered_docs, file_paths
+):
+    """Atualiza article-id (scielo-v2 e scielo-v3) dos documentos recebidos."""
```

Comment:
@robertatakenaka 

Não devemos registrar quando a versão for v1?

Reply:
Este issue surgiu na necessidade de o site novo conseguir linkar com o site atual. Então, acho que não entraria a versão 1.

Reply:
@robertatakenaka 

Quando vc se refere a "**linkar**" isso significa consegui resolver as URL do **site atual** é isso?

Reply:
sim. Mas sua pergunta sobre a versão 1 é pertinente.

Reply:
@jamilatta no entanto, pensando melhor, essa modificação fica no converter e portanto, estamos lidando com documentos mais recentes.
Seria o caso de incluir o scielo pid da versão 1 na migração.

Reply:
@robertatakenaka 

Então devemos abrir um ticket na migração sinalizando essa necessidade? 

No meu questionamento inicial a minha idéia era justamente essa, em algum lugar devemos obter o **scielo-v1**. 

Me confirme @robertatakenaka para que possamos abrir um tíquete para não esquecermos!

Reply:
isso

Reply:
Ok, irei criar um tíquete para essa demanda no projeto de migração!

Reply:
@robertatakenaka 

Criado o tíquete: https://github.com/scieloorg/document-store-migracao/issues/215